<launch>
  <arg name="team" default="1"/>
  <group ns="robot$(arg team)">
    <group>
      <node name="find_destination" pkg="player" type="find_destination_server.py" />
      <node name="move_to_destination" pkg="player" type="move_to_destination_server.py" />
      <node name="player_sm" pkg="player" type="player_sm.py" />
      <node name="object_detector" pkg="player" type="object_detector.py" />
    </group>
  </group>
</launch>